### ISSUE TITLE
assistive technology software is unable to detect the object tags

### ISSUE THREAD

**Issue by OlcayAshchi** [deltakosh/Podcasts#26]
the controls cannot be interpreted by screen readers that make up objects in the interface. example, where there is now playing uwp object the object are performed. mouse over it to read the text to read correctly from the front. texts written on the labels of the objects found in the properties of the objects on the form, accessibility should be written on the label. although the application is accessible, this error makes it difficult to use the application.   arayüzdeki kontrolleri oluşturan nesneler ekran okuyucular tarafından yorumlanamıyor. örneğim, şimdi yürütülüyor nesnesini uwp nesnesi olduğu seslendiriliyor. bunun doğru okunması için on mouse over metninin okunması gerekiyor. nesnelerin etiketlerinde yazılı metinler, form üzerinde nesnelerin özelliklerinde bulunan, accessibility label üzerine yazılmalı. uygulama erişilebilir olmasına rağmen bu hata uygulamayı kullanmayı zorlaştırıyor.
